src/backend/impress/settings.py

```diff
@@ -680,6 +681,21 @@ class Base(Configuration):
         },
     }
 
+    MALWARE_DETECTION = {
+        "BACKEND": values.Value(
+            "lasuite.malware_detection.backends.dummy.DummyBackend",
```

Comment:
I'm not sure I like this, because by default, file will be marked as "safe" which is not the case.

Reply:
We can only provide the dummy backend by default. JCOP needs credentials.

Reply:
Ok, since the "analyze status" is not stored anywhere this is not an issue: the status is only used by the callback to upload the file. I guess you can keep it like this then :)